### DISCUSSION TITLE
Welcome to understrap-child Discussions!

### DISCUSSION THREAD

**Discussion by danemorgan** [understrap/understrap-child#280] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nicholasdixon** [comment]
Hi

I am wanting to learn how to do the following type of customisations.

I am wanting to develop the understanding to tinker with the code to add a banner image (from a photograph)above and below the navigation. I would want to code the child theme so I can update the banner from inside the customiser. 

I also would like to know is it possible to have a navigation along the top of the page, then a banner image (from a photograph) about 80pixels high, and fluid width with a secondary navigation below.

I have had a go at doing this myself but got errors. 

Can you give me some advice on how to achieve this.

**Reply by danemorgan** [reply]
What is your knowledge level for HTML, CSS, and PHP?

**Comment by nicholasdixon** [comment]
Hi, I am about intermediate level in css and html, I have done several Wordpress courses including theme development on LinkedIn learning and team treehouse, also I did the php for Wordpress course on treehouse. How ever I am not a expert.

Sent from my iPhone

On 2 Aug 2021, at 02:36, Dane Morgan ***@***.***> wrote:

﻿

What is your knowledge level for HTML, CSS, and PHP?

**Reply by 0dp** [reply]
I'd recommend that you try and post the code in a step by step manor that you have made yourself and also post any errors you are receiving.
I think that'll be a good starting point.

**Comment by nicholasdixon** [comment]
I deleted the code, but will have another go and post the code in steps, as soon as I can.

Sent from my iPhone

On 2 Aug 2021, at 12:39, Johan Nielsen ***@***.***> wrote:

﻿

I'd recommend that you try and post the code in a step by step manor that you have made yourself and also post any errors you are receiving.
I think that'll be a good starting point.

**Comment by nicholasdixon** [comment]
******THE ANSWER********

I had another go and this time it seemed to work.
For some reason Github wouldn't allow me to put php tags before the header image tag, so I tried to re-posted it a couple of times and then deleted it. So apologies Dane and Odp if you got a few messages.

I ADDED THE FOLLOWING TO THE BOTTOM OF THE functions.php FILE.

function understrap_child_custom_header_setup() {
    add_theme_support( 'custom-header', apply_filters( 'understrap_child_starter_custom_header_args', array(
        'flex-width'             => false,
        'default-image'          => '',
        'default-text-color'     => 'fff',
        'width'                  => 1000,
        'height'                 => 250,
        'flex-height'            => false,
        'wp-head-callback'       => '',
    ) ) );
}
add_action( 'after_setup_theme', 'understrap_child_custom_header_setup' );


I ADDED THE TAG WHICH DISPLAYS THE CUSTOM HEADER TO BEFORE THE MENU, IT ALSO ADDS A CLASS FOR IT.
FOR SOME REASON GITHUB WOULDNT ALLOW ME TO PUT PHP TAGS INFRONT OF THE FOLLOWING TAG, IT CAUSED A DISPLAY ISSUE FOR THE CODE ETC.
the_header_image_tag( array( 'class' => 'my-img-fluid' ) ):


I ADDED THE FOLLOWING CSS IN THE child_theme.scss:
my-img-fluid {
	width: 100%;
	height: auto;
}

THIS MAKES THE BANNER RESPONSIVE.

**Reply by danemorgan** [reply]
So, does this mean it is working now?

**Reply by danemorgan** [reply]
Hi @nicholasdixon  

Would you mind accepting this as the answer to help anyone who comes along and finds this later down the line?

**Reply by nicholasdixon** [reply]
HI Dane sorry for the confusion, yes I got it working. And I accept it as the answer, however I cannot find the option to formally accept anything as an answer on Github discussions.

**Reply by bacoords** [reply]
Since this conversation is technically a response to another issue (the welcome post) we can't accept it as an answer. No worries!
